examples/hackernews/reverse_proxy/haproxy.cfg

```diff
@@ -4,35 +4,39 @@ frontend www
     bind :443 ssl crt /etc/ssl/selfsigned.crt
     http-request redirect scheme https unless { ssl_fc }
 
-
-    # Redirect /streams/... to the Skip streaming backend.
-    acl url_streams path_beg -i /streams/
-    use_backend stream if url_streams
-
-    # Redirect /api/... to the api backend.
+    # Dispatch /streams/$follower/$uuid to $follower
+    acl url_follower1 path_beg -i /streams/1/
+    acl url_follower2 path_beg -i /streams/2/
+    acl url_follower3 path_beg -i /streams/3/
+    use_backend follower1 if url_follower1
+    use_backend follower2 if url_follower2
+    use_backend follower3 if url_follower3
```

Comment:
Would something like that work (provided the env var given to the followers match the follower names)?
```suggestion
    # Dispatch /streams/$follower/$uuid to $follower
    use_backend %[path_beg,field(2, '/')]
    default_backend follower1
```

Reply:
I made it this way to avoid leaking backend names -- trying it out that way now to see if it works for people who don't care about leaking that bit of data.

I think a better endstate is using Haproxy maps to map external-facing identifiers (`1`, `2`, `3` here) to actual backend names.  That would have the additional benefit of being elastic using the runtime API's set-map/del-map operations.

Reply:
The suggested `use_backend` incantation requires that the external-facing path looks like `/streams/<backend_name>/...`, i.e. includes the actual name of the backend. (`follower1`, `follower2`, etc. here)

This version has one layer of indirection, with an external-facing identifier (`1`, `2`, `3`) that doesn't need to match the actual haproxy backend name.  Probably a minor concern in the scheme of things.